ariadne/cardano/src/Ariadne/Wallet/Backend/KeyStorage.hs

```diff
@@ -221,15 +221,18 @@ newAddress ::
        PassiveWalletLayer IO
     -> WalletFace
     -> IORef (Maybe WalletSelection)
+    -> (WalletReference -> IO PassPhrase)
+    -> (WalletReference -> IO HdAddress -> IO HdAddress)
+    -> WalletReference
```

Comment:
Why did this argument become needed here? `AccountReference` is already passed here and AFAIU it can be converted to `WalletReference`.

Reply:
Did not know `WalletReference` could always be obtained from an `AccountReference`, removed it from the function and also reverted some code in `Knit` that was changed for this same reason